build.gradle.kts

```diff
@@ -11,6 +11,7 @@ plugins {
     `maven-publish`
     id("io.github.gradle-nexus.publish-plugin")
     signing
+    id("org.octopusden.octopus-release-management")

```

Comment:
Please add the comment for the puprose of this plugin

Reply:
Scenario?

Reply:
The published artifact will be used by the `CreateTeamCityBuildChain` meta-runner, allowing it to be directly used in the build configuration.

Reply:
The published artifact will be used by the `CreateTeamCityBuildChain` meta-runner, allowing it to be directly used in the build chain configuration.

Reply:
@adellinekania, please check whether we can avoid using it.

Reply:
Decided to remove it